Opserver/Views/Dashboard/Node.Services.cshtml

```diff
@@ -9,6 +9,10 @@
             <th>Service</th>
             <th>Description</th>
             <th>State</th>
+            @if (Current.IsInRole(StackExchange.Opserver.Models.Roles.GlobalAdmin))
+            {
+            <th>Action</th>
+            }
```

Comment:
We need to make this conditional, for anyone *not* using WMI we're putting a button they can't use.

I think what I'd do here is instead of putting service controls on the base dashboard provider, add an interface...I'll leave a comment there. Here, we'd do:
```c#
@if (Current.IsInRole(StackExchange.Opserver.Models.Roles.DashboardAdmin) && Model.HasServiceControl())
```
This interface being present on the provider would let you know if it implements controls.